packages/formatting/src/utils/detection.ts

```
@@ -0,0 +1,198 @@
+import { exec } from 'node:child_process';
+import { promisify } from 'node:util';
+import { access, constants } from 'node:fs/promises';
+import { join } from 'node:path';
+import type {
+  FormatterDetection,
+  FormatterDetectionResult,
+  FormatterType,
+} from '../types/index.js';
+import type { Result } from '@outfitter/contracts';
+import { success, failure, makeError } from '@outfitter/contracts';
+import { FormatterDetectionSchema, FormatterDetectionResultSchema } from '../schemas/index.js';
```

ESLint

[error] 'FormatterDetectionSchema' is defined but never used.

(@typescript-eslint/no-unused-vars)